src/app/Services/Generator.php

```diff
@@ -18,7 +18,7 @@ public function __construct(Choices $choices)
     public function handle()
     {
         if ($this->isNotConfigured()
-            || ($this->choices->needsValidation() && $this->failsValidation())) {
+            || $this->choices->needsValidation() && $this->failsValidation()) {
```

Comment:
It may a little confusing, isn't it good to wrap it with parentheses?

Reply:
logical operators precedence should be pretty easy to parse. In this specific case we even split the code on two distinct lines.

Reply:
![image](https://user-images.githubusercontent.com/10864136/71644673-a5413800-2ce1-11ea-89dd-fa0930aaf8a5.png)
my IDE shows this line as an error because it tells it's hard to read :-D

Reply:
now your IDE should be fine :D

Reply:
it's much nicer now :))))